test/expected/create_hypertable.out

```diff
@@ -974,3 +974,28 @@ ERROR:  insert or update on table "_hyper_21_21_chunk" violates foreign key cons
 DROP TABLE regular cascade;
 NOTICE:  drop cascades to 2 other objects
 DROP TABLE timescale cascade;
+-- github issue 4872
+create table some_hypertable (
```

Comment:
can we have a better name here instead of some_hypertable, eg tidrangescan_test, also the test gives no context what it is about

Reply:
Fixed.